pkg/raft/opts.go

```diff
@@ -126,6 +127,31 @@ func (this *options) ReadOption() raft.ReadOnlyOption {
 	return raft.ReadOnlySafe
 }
 
+func getLocalIPAddress() ([]net.IP, error) {
```

Comment:
Consider returning a `map[string]net.IP` so that checking for membership can be done in constant time.